### PR TITLE
🔀🔖 4.3.5

### PR DESCRIPTION
**Pull request by jxn-30**
* 🚑️ [ecw/mk] fix 0 credits issue
* 🐛 [redesign/vehicle] show foam_amount if present
* 💄🐛 [ge/ms] fix chrome issue
* 🌐 [emv] fix several issues in multiple languages
* 🌐🐛🚀 [ep] add some missing pois in several languages
* 🚸 [ge/cl] remove img on error
* 🚸 [releasenotes] reduce cache problems
* ✨🇳🇴🔥🚒 support New Content Pack 🚒🔥🇳🇴✨
* ✨📝 [docs] improve moment.js config docs
* ✨🚸🐛 allow multiple schoolings on vehicle
* ✨ [blf] toggle all button
* ✨ [ecl] remaining time for patients & pumping
* ✨ [redesign] add verband/kasse
* ✨ new module `extendedMap` with features:
    * mapScale
    * centerMap (static mode)

### CODE REVIEW

**Review by Ron31 — APPROVED** [review]

**Review by Suchty112 — APPROVED** [review]

File: docs/de_DE/settings.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ lang: de_DE
 In den Einstellungen werden die Einstellungen aller Module zentral verwaltet. Es können nur die Einstellungen aktiver Module verändert werden.
 
 Mittels der Knöpfe `Export` und `Import` kannst du deine Einstellungen in einer Datei sichern und / oder an deine Freunde weitergeben.
-Wir möchten in naher Zukunft die Möglichkeit bieten, Einstellungen Profilgebunden zu speichern. Damit sind Einstellungen nichtmehr Gerätegebunden.
+Wir möchten in naher Zukunft die Möglichkeit bieten, Einstellungen Profilgebunden zu speichern. Damit sind Einstellungen nicht mehr Gerätegebunden.
```
Comment (Cr4zyc4k3) [inline docs/de_DE/settings.md:11]: gerätegebunden wäre grammatikalisch korrekt. Würde das aber umformulieren in "an das Gerät gebunden" bzw. "an ein Gerät gebunden" oä

File: docs/nl_NL/settings.md [hunk 2]
```diff
@@ -18,3 +18,26 @@ Als u niet-opgeslagen wijzigingen heeft, kunt u de instellingen niet sluiten, u
 ::: danger Reset de instellingen
 Let op: Als u instellingen reset, kunnen ze niet worden hersteld zonder voorafgaande export!
 :::
+
+
+## Moment.js
+This section provides some information on how to configure the date-time configs, e.g. in the module [clock](modules/clock.md).
```
Comment (Cr4zyc4k3) [inline docs/nl_NL/settings.md:24]: English in the dutch version?

File: typings/helpers.d.ts [hunk 3]
```diff
@@ -45,7 +49,7 @@ declare global {
             leafletMissionPositionMarkerAdd(poi: POI): void;
         };
         map_filters_service: {
-            initialize(a: MapFilterInitializer): void;
+            // initialize(a: MapFilterInitializer): void;
```
Comment (Cr4zyc4k3) [inline typings/helpers.d.ts:52]: Remove if not needed

File: docs/nl_NL/settings.md (hunk 2 above)
Comment (jxn-30) [reply docs/nl_NL/settings.md:24]: @JRH-1997 is already informed about missing NL translations ;)

File: typings/helpers.d.ts (hunk 3 above)
Comment (jxn-30) [reply typings/helpers.d.ts:52]: commented because will be re-needed later for centerMap dynamic mode

**Review by Cr4zyc4k3 — APPROVED** [review]

File: docs/de_DE/settings.md (hunk 1 above)
Comment (jxn-30) [reply docs/de_DE/settings.md]: => d6e346d

**Review by Cr4zyc4k3 — APPROVED** [review]

**Pull request merged**